### DISCUSSION TITLE
Why so few Youtube videos?

### DISCUSSION THREAD

**Discussion by npearson72** [mantinedev/mantine#2417] [Q&A]
I just recently stumbled across Mantine. I have to say it's a very impressive library. I've been looking for something more robust than Chakra but less heavy than MUI.

So first off, thank you. 

My question though is why with 14.2k stars there are so few Youtube videos on Mantine? When I look up information on Youtube I'm overwhelmed by Tailwind, MUI, Bootstrap, etc. But no Mantine. There are maybe 3 channels that I've come across discussing Mantine.

Now I know some of you might wonder why I'm not helping in that regard; and it's because I have a face and voice only a mother could love. So I stay away from making videos.

But seriously, it's such a great library... why the poor marketing?

**Comment by Blazing-Mike** [comment]
@npearson72 I'm not in the Mantine team but I agree with you, some tools don't put a lot of effort into marketing as they believe the product would market itself if it is good or maybe they don't just see it yet or they have plans for it in the future. 

I came across Mantine on Fireship's YouTube video and I must say I'm impressed. I thought it would be just like every other hyped and shiny new thing but this is really different.  That being said what kind of videos resources would you suggest be created for a start I'm willing to help.

**Reply by npearson72** [reply]
Hi @Blazing-Mike, thanks for replying. I didn't expect anyone to actually respond. I was just curious about this situation given that MUI, Chakra, and Tailwind (albeit Tailwind is a different paradigm than the rest) get so much hype.

I really took a deep dive into studying the different options and was leaning toward headless (not necessarily headlessui, but "headless" in general). Was taking a close look at Adobe's React Aria library.

But in the end, I decided headless demands too much of my time at this stage of my project.

I wasn't thrilled about MUI because it seems like they themselves are confused about what they want to offer. They have the main library, then a headless version (MUI Base), and now some new thing called Joy UI.

The main library has a ton of components, but honestly the layers of wrapping and class names and styling systems seems over-architected to me. I also don't like that I have to wrestle with disabling that Material look before I can achieve the look I want.

I then looked at Chakra UI. It seemed promising but not very functional.

I spotted one video (just one) about Mantine on Youtube and decided to take a look. At first I figured it was "yet another UI library", but as I studied it, I realized it:

- Uses all of today's best practices (CSS in JS via Emotion),
- Has a tone of components
- Offers a very intuitive and flexible theming system
- Is very feature rich with a hooks library
- Is cleanly implemented (less is more)

Overall I was just really surprised at how good it was; which got me thinking... why haven't I heard about Mantine earlier? It would have saved me a lot of time studying the other hyped up options.

So to answer your question about the type of Youtube videos: I think anything will do. So long as there are enough of them that the algorithm promotes Mantine such that people hear about it.

**Reply by Blazing-Mike** [reply]
Ok makes sense. I'll try to do a video using to build something small. I also think it is matter of time before it gets popular and I also found a few YouTube videos about it.

**Comment by 7iomka** [comment]
I think the main problem with the unpopularity of the library is that the ideas taken as a basis, which distinguish this library from any others at first, were extremely experimental and a huge number of breaking changes came even in ordinary minor updates, so give a video on YouTube showing implementation of "how to do something there" would not be actual in a week.

When I looked closely at the library, I saw 2nd, 3rd, and 4th version, and v5 was in alpha, in which all the bugs were fixed that were in version 4 and which were critical for me to take the library.
And what do you think? I took the alpha version into a real project, and made the right decision - after a couple of weeks, support for all old versions stops, moreover, the documentation for them was simply removed from the site (users who sat on them would have to use it only by downloading from github and running locally).
The library itself is revolutionary, but it is run by just a couple of brilliant people, in the interests of which to create something ingenious and for all occasions in the shortest possible time, and issues of supporting outdated ideas are left at the mercy of users.
If you are ready to participate in such a race, welcome, I believe that over time the library will become more stable in terms of breaking updates)

**Reply by npearson72** [reply]
I didn't know that. Is the story much different with other libraries?

**Reply by 7iomka** [reply]
This is not (only) my opinion - this is an opinion I heard in a telegram chat where people were interested in which component library they would recommend in 2022. And they offered mantine, with a postscript that even minor updates can have breaking changes and if you are not ready to support the codebase along with updates - the library will not suit you.

**Reply by npearson72** [reply]
Well that settles it then, doesn't it.

**Comment by michael-azogu** [comment]
Because it's a side project not a real organization, the core team isn't really a core team it's just vitaly .

**Reply by npearson72** [reply]
Kudos to @rtivital  if he's able to build a better library than most of the ones I've seen out there while doing it on the side no less.

Still though, most of the Youtube videos I see aren't made by the developers, but rather vloggers and the like.
